src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
Yes, the printer for list is buggy, it also prints a single-element list as a string, which is misleading for the user (i.e. `opam init` currently prints `backup_urls` as a string...).
I don't really know how to do what you say: it would involve reprinting the `config` file after adding a new field `backup_urls`, but then, to do that, I need to change the current global version of OPAM, since the config file does not have its own version, just the global version of OPAM...